docs/Documentation/Development/VSCode/index.md

```diff
@@ -77,3 +77,31 @@ Now use the Remote-SSH extension in VS Code to `Connect to Host...` and use the
 
 This should open a new VS Code window that will connect to the compute node automatically. You may begin browsing your home directory and editing files in the VS Code window.
 
+## Jupyter Notebook in VS Code
+Before proceeding with this section, please read [Interactive Parallel Python with Jupyter](../Languages/Python/KestrelParallelPythonJupyter/pyEnvsAndLaunchingJobs.md) as this document will be using its process as a reference.
+
+### Setting Up VS Code
+To begin, proceed to VSCode and install these additional extensions, if you do not already have them: [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python) and [Jupyter](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter).
+
+### Setting Up Conda Environment
+In addition, you will also need to set up a Python Environment on Kestrel as per the [Interactive Parallel Python with Jupyter](../Languages/Python/KestrelParallelPythonJupyter/pyEnvsAndLaunchingJobs.md#setting-up-your-account) documentation. Please note that you do not need a python environment on your local machine for this. 
+
+### VS Code Jupyter on Kestrel
+
+With the Conda environment set up, the process will be following [multinode Jupyter job example](../Languages/Python/KestrelParallelPythonJupyter/pyEnvsAndLaunchingJobs.md#multinode-capable-job-eg-mpi4py-through-ipyparallel) through VS Code on the compute. Download the [example Jupyter script](../Languages/Python/KestrelParallelPythonJupyter/exampleNotebooks/cupyAndIpyparallel.ipynb) onto Kestrel to load it into VS Code later.
+
+### Intializing Environment
```

Comment:
The steps for startinga. job and connecting to VS code are documented above on this page, so that can be referenced. I think all that is needed is to explain how to select the Python Kernel for the environment